plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -2000,5 +2002,58 @@ var _ = Describe("validator", func() {
 				Expect(err).To(HaveOccurred())
 			})
 		})
+
+		Context("shoot deletion", func() {
+			var (
+				shootStore             cache.Store
+				intCoreInformerFactory intCoreinformers.SharedInformerFactory
+			)
+
+			BeforeEach(func() {
+				intCoreInformerFactory = intCoreinformers.NewSharedInformerFactory(nil, 0)
+				shootStore = intCoreInformerFactory.Core().InternalVersion().Shoots().Informer().GetStore()
+				Expect(coreInformerFactory.Core().InternalVersion().CloudProfiles().Informer().GetStore().Add(&cloudProfile)).To(Succeed())
+				Expect(coreInformerFactory.Core().InternalVersion().Seeds().Informer().GetStore().Add(&seed)).To(Succeed())
+				Expect(coreInformerFactory.Core().InternalVersion().Projects().Informer().GetStore().Add(&project)).To(Succeed())
+			})
+
+			It("should reject if the shoot has lastOperation=Migrate", func() {
```

Comment:
This and the other tests looks very similar, only the last operation type is different.
So, you can write them in `DescribeTable` to avoid code duplication.